doc/contributing/reviewing-contributions.xml

```diff
@@ -515,19 +515,19 @@
    It is possible for community members that have enough knowledge and experience on a special topic to contribute by merging pull requests.
   </para>
 
-  <para>
-   TODO: add the procedure to request merging rights.
-  </para>
-
 <!--
-The following paragraph about how to deal with unactive contributors is just a
+The following paragraphs about how to deal with unactive contributors is just a
 proposition and should be modified to what the community agrees to be the right
 policy.
 
 <para>Please note that contributors with commit rights unactive for more than
   three months will have their commit rights revoked.</para>
```

Comment:
While you're here, it might be worth mentioning https://github.com/NixOS/rfcs/blob/master/rfcs/0055-retired-committers.md - so the policy would be something along the lines of "Please note that per <link xlink:href="https://github.com/NixOS/rfcs/blob/master/rfcs/0055-retired-committers.md">RFC #55</link>, contributors with commit rights to nixpkgs who are inactive for more than a year will become retired contributors."

...but that's also possibly worth a separate CL.

Reply:
Happy to integrate that in here since we didn't quite reach a conclusion.
Also, as you mentioned RFC, I'm wondering if we should morph the current discussion into one.
Any comments on this from the more active members following this discussion?

Reply:
@worldofpeace @cole-h I'm keen for this PR not to stall - any input on the above?